core/src/com/unciv/ui/screens/overviewscreen/WonderOverviewTable.kt

```diff
@@ -44,7 +44,8 @@ class WonderOverviewTab(
 
         top()
         defaults().pad(10f).align(Align.center)
-        (1..5).forEach { _ -> add() }  // dummies so equalizeColumns can work because the first grid cell is colspan(5)
+        for (i in 1..5)
```

Comment:
HAH so your linter agrees with me that for > forEach! #validated

Reply:
Yep. So does this: https://sites.google.com/a/athaydes.com/renato-athaydes/posts/kotlinshiddencosts-benchmarks

`The above graph shows that using the forEach function for a Range, as the author predicted, should absolutely be avoided. It performs 300% slower than a simple for-loop!`